apps/src/applab/VisualizationOverlay.js

```diff
@@ -121,7 +124,16 @@ VisualizationOverlay.prototype.onSvgMouseMove_ = function (event) {
 
   this.mousePos_.x = event.clientX;
   this.mousePos_.y = event.clientY;
-  this.mousePos_ = this.mousePos_.matrixTransform(this.screenSpaceToAppSpaceTransform_);
+  var draggingElement = $(".ui-draggable-dragging");
```

Comment:
I'm fine with this for now - this class is already acting as glue code between React and non-React worlds.

Nit: I'm not sure whether `draggingElement` is supposed to be a boolean or jQuery wrapping a single element, but in reality it's neither - it's a selection.  I'd prefer one of:
```javascript
var isDraggingElement = !!$(".ui-draggable-dragging").length;
// or
var draggedElement = $(."ui-draggable-dragging").first();
```